packages/charts/src/chart_types/heatmap/layout/types/config_types.ts

```diff
@@ -28,6 +28,15 @@ import { Cell } from './viewmodel_types';
 export interface Config {
   width: Pixels;
   height: Pixels;
+  /**
+   * Override the computed left and right overflow margin
+   * Helpful if fixed positioning is required
+   * e.g. ending right position will be overall width - xAxisOverflow.right
+   */
+  xAxisOverflow?: {
+    right?: number;
+    left?: number;
```

Comment:
at the moment the user can provide `yAxisLabel` config with fixed `width` and `padding`, so I'm not sure if we need it here 🤔